### PR TITLE
Logo in Readme

### LINKED ISSUE #21: Add Beautiful Logo to README
**User Story**
As a User, I would like to see the logo that exists for the project to be displayed in the README for the project.

**Acceptance Criteria**

- [x] Add the logo to the README 

**Why is this feature needed? Please describe the problem your requested feature wants to solve**
We have a beautiful logo, and it would be pointless if we don't use it somewhere. Adding it to the README would be a good start.

**Describe the solution you'd like**
Add a section to the README that displays the beautiful logo we have.

### PR DESCRIPTION
**Pull request by KimberleyEvans-Parker**
GitHub Issue (If applicable): #21 

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behaviour?
Readme has no logo

## What is the new behaviour?
Readme has 'Roll for Reaction's logo

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [ ] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [ ] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information

Internal Issue (If applicable):
closes #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Roll For Reaction
+
+<img src="https://github.com/Matteas-Eden/roll-for-reaction/blob/master/public/logo.png" width="400">
+
```
Comment (Matteas-Eden) [inline README.md]: I'd prefer it if we used a link that isn't subject to change, like the one posted in a comment on the issue.

`logo`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Roll For Reaction
```
Comment (Matteas-Eden) [inline README.md]: If we have the logo (which contains the title of the project), I don't think this heading is necessary anymore.

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
Looks good, but I've left some feedback I'd like you to look at.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-# Roll For Reaction
+<img src="https://user-images.githubusercontent.com/45865186/77014224-a9265180-69d6-11ea-92d3-18af1d76c740.png" width="400">
+
```
Comment (Pyxxil) [inline README.md]: Why are we not using relative links? This is the recommended way to do images in markdown.

**Review by Pyxxil — DISMISSED** [review]

**Review by Matteas-Eden — DISMISSED** [review]
LGTM, thanks for addressing my feedback.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
-# Roll For Reaction
+<p align="center">
+<img src="https://user-images.githubusercontent.com/45865186/77014224-a9265180-69d6-11ea-92d3-18af1d76c740.png" width="400">
```
Comment (Matteas-Eden) [inline README.md]: Change to use relative link to wider image:
`<img src="public/logo-wide.png"`

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
Left some feedback in accordance with discussion on Messenger.

**Review by Matteas-Eden — APPROVED** [review]
It's... *beautiful*.

**Pull request merged**